e2/electron.vite.config.ts

```diff
@@ -12,6 +13,6 @@ export default defineConfig({
     plugins: [externalizeDepsPlugin()]
   },
   renderer: {
-    plugins: [nodePolyfills({include: ['buffer'], globals: {global: false, process: false}}), svelte()]
+    plugins: [nodePolyfills({include: ['buffer'], globals: {global: false, process: false}}), svelte(), lingui({configPath: "../app/lingui.config.ts"})]
```

Comment:
Could you please make newlines after every plugin, similar to app/vite.config.js ?